DBugVision/constants.py

```diff
@@ -0,0 +1,6 @@
+# Video Stream Constants
+CAMERA_INDEX = 0 # The camera device index (in the OS) 0 is for the default camera chosen by the operating system and OpenCV

```

Comment:
If this is a default value - it should be the default value of it's appropriate argument and with an appropriate name:
i.e: ```def __init__(self, camera_device_index=constants.DEFAULT_CAMERA_INDEX):```